src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -41,7 +41,7 @@
 #include "rest/httpRequestSend.h"
 #include "ngsiNotify/senderThread.h"
 #include "ngsiNotify/Notifier.h"
-
+#include "cache/subCache.h"
```

Comment:
Is this really needed? Maybe it was during the work in this PR due to a part that now is missing... in the current diff nothing new seems to be added to this file that needs this include.